templates/00-hello-world/src/local_test_environment.js

```diff
@@ -0,0 +1,82 @@
+const NodeEnvironment = require('jest-environment-node');
+const dev = require('nearlib/dev');
+const fs = require('fs');
+const nearlib = require('nearlib');
+
+
+class LocalTestEnvironment extends NodeEnvironment {
```

Comment:
why not ship this as part of nearlib package which isn't compiled for browser?

Reply:
This is now a part of shell new project. I can use this PR to bring this in